### ISSUE TITLE
Consider applications - odometry example

### ISSUE THREAD

**Issue by AWatk** [vanderbiltrobotics/covariance-example#3]
Once it is packaged all nice like for importing elsewhere, you're ready to start applying covariance tracking to more specific examples.  One huge huge point of data that needs to be refined and as reliable as possible is your odometry.

See *Introduction to Autonomous Mobile Robots* by *Siegwart and Nourbakhsh*, **Chapter 5**, "An Error Model for Odometric Position Estimation"
[odometry_model.pdf](https://github.com/vanderbiltrobotics/covariance-example/files/1788057/odometry_model.pdf)

**Comment by SwapnilPande** [comment]
**Chapter 5.3-5.4** of _Probabilistic Robotics_ also covers Odometry & Velocity based motion models

**Comment by AWatk** [comment]
Yep! The hardest part about those motion models are the number of robot-specific parameters (alpha 1-6).

I haven't had time to look into it, but possibly the odometric position error model for a differential drive robot can help derive those robot-specific parameters? Or does it appear to be two separate methods of doing the same thing?
